docker-compose.yml

```diff
@@ -17,43 +16,31 @@ services:
       - P2P_ENVIRONMENT=${P2P_ENVIRONMENT}
       - P2P_DISCOVERY_NAMESPACE=${P2P_DISCOVERY_NAMESPACE}
     ports:
-      - "8080:8080"
```

Comment:
Docker exposes all mapped ports publicly. We don't want that on the live server.